include/deal.II/fe/fe_values.h

```diff
@@ -3203,11 +3203,10 @@ namespace FEValuesViews
   Scalar<dim,spacedim>::value (const unsigned int shape_function,
                                const unsigned int q_point) const
   {
-    typedef FEValuesBase<dim,spacedim> FVB;
     Assert (shape_function < fe_values->fe->dofs_per_cell,
             ExcIndexRange (shape_function, 0, fe_values->fe->dofs_per_cell));
     Assert (fe_values->update_flags & update_values,
-            typename FVB::ExcAccessToUninitializedField("update_values"));
+            ((typename FEValuesBase<dim,spacedim>::ExcAccessToUninitializedField("update_values"))));
```

Comment:
I did not. I accidentally added a second set to all such parentheses at one point and then I deleted (nearly all) of them: I must have missed this pair.